### ISSUE TITLE
Default setup and model file not valid

### ISSUE THREAD

**Issue by luke-hale** [imartinez/privateGPT#940]
**Describe the bug and how to reproduce it**
Hey, I am using the default model file and env setup. The model file is not valid.

**Expected behavior**
Running `python3 privateGPT.py` - expect to be able to input prompt. Model file is not valid (I am using the default mode and Env setup). Ingest.py works as expected.

```
Found model file at  models/ggml-gpt4all-j-v1.3-groovy.bin
Invalid model file
Traceback (most recent call last):
  File "/Users/luko/Documents/privateGPT/privateGPT.py", line 83, in <module>
    main()
  File "/Users/luko/Documents/privateGPT/privateGPT.py", line 38, in main
    llm = GPT4All(model=model_path, max_tokens=model_n_ctx, backend='gptj', n_batch=model_n_batch, callbacks=callbacks, verbose=False)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/langchain/load/serializable.py", line 74, in __init__
    super().__init__(**kwargs)
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for GPT4All
__root__
  Unable to instantiate model (type=value_error)
```

**Environment (
 - OS / hardware - macOS 13.4.1  / M1 Max 
 - Python version -3.11.3 

New to Python - thanks so much!

Env setup (copied from example.env): 
```
PERSIST_DIRECTORY=db
MODEL_TYPE=GPT4All
MODEL_PATH=models/ggml-gpt4all-j-v1.3-groovy.bin
EMBEDDINGS_MODEL_NAME=all-MiniLM-L6-v2
MODEL_N_CTX=1000
MODEL_N_BATCH=8
TARGET_SOURCE_CHUNKS=4
```

**Comment by ASamuli** [comment]
Hi, I did have to downgrade gpt4all via pip to get it working. This is what might work to this since it works on my machine and I did read about it, when trying to get it work in an older post, I would refer to it if I could find it. I can remember downgrading gpt4all worked to similar issue, but not sure it's the exact same error.

I use currently below gpt4all version with privateGPT:
Name: gpt4all
Version: 0.2.3

Hope it helps.
